README.md

```diff
@@ -62,24 +62,84 @@ please make one and assign yourself.
    the 'Review checklist'. Leave any notes for the reviewer in the 'Details'
    section.
 
-4. Mark the pull request as ready for review, and assign @stuartc or
-   @taylordowns2000 .
+4. Mark the pull request as ready for review, and assign @stuartc or @taylordowns2000.
 
 ### Working with adaptors
 
-To create a new adaptor follow the steps below
+#### 1. Setup 
 
 - Create a copy of the adaptor [template](packages/template), and rename it.
-  `cp packages/template packages/newadaptorname`
+  `cp -R packages/template packages/youradaptorname`
 
-- Define the configuration schema
+- Open package.json, and update the `name`, `description` and `build` from `template` to your `youradaptorname`. Update the version.
 
-  The configuration schema defines required parameters and their expected values
+- Update the top-level changelog comment `# @openfn/language-template` to your adaptor name and delete the example content. Delete ast.json, then run `pnpm clean` (this will remove the dist, types and docs folders). 
+
+- Update your adaptor's README.md to use your new adaptor name (we'll come back to this later to update the sample operation)
+
+#### 2. Create a job with the operation you'd like to add, and run it so it fails
+
+- Create a _temporary_ folder where you can test your operation manually by running a job: `mkdir tmp`
```

Comment:
I wouldn't emphasis _temporary_, it makes it look like you have to do something special. Like, how do I create a  temporary folder?

Probably important to note that the contents of `tmp` don't get checked in to github - so this folder name isn't arbitrary, it's a convention we use.